suite-common/connect-init/src/connectInitThunks.ts

```diff
@@ -124,19 +124,11 @@ export const connectInitThunk = createThunk(
                 _sessionsBackgroundUrl = null;
             } else if (window.location.origin.endsWith('dev.suite.sldev.cz')) {
                 // we are expecting accompanying connect build at specified location
-                const assetPrefixArr = (process.env.ASSET_PREFIX || '').split('/').filter(Boolean);
-                const relevantSegments = assetPrefixArr
-                    .map((segment, index) => {
-                        const first = index === 0;
-                        const last = index === assetPrefixArr.length - 1;
-                        if (segment === 'suite-web' && first) return 'connect';
-                        if (segment === 'web' && last) return null;
-
-                        return segment;
-                    })
```

Comment:
are you sure that we don't need this segment? I already had this file marked as "viewed" and now this change appeared here again I think

Reply:
I am getting this again. feels related
<img width="1003" alt="image" src="https://github.com/user-attachments/assets/3c56ae26-ce6e-47cd-9002-d30d720a3a6a">

Reply:
Yes, you are right, that came back as a result of the rebase, I missed it, but it is back now it is working as expected.